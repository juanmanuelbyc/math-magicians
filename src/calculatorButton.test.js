import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorButton from './components/calculatorButton';

const clickHandler = (button) => {
  const newState = calculate(obj, button);
  setObj(newState);
};

it('renders correctly', () => {
  const tree = renderer.create(
    <CalculatorButton type="characterButton" size="s1" label="AC" handleClick={clickHandler} />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
