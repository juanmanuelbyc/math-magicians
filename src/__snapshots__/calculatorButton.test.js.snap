// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="s1 characterButton button"
  onClick={[Function]}
  type="button"
>
  AC
</button>
`;
